doc/vxlan/EVPN/EVPN_VxLAN_Multihoming.md

```diff
@@ -0,0 +1,1821 @@
+Remember, any traffic sent by the host may end up on the other homing TOR based on the bundle-Ethernet interface hashing. This is where MAC, ARP and ND table synchronization come to play. All learned entries (local adjacencies) on a TOR are synchronized to all other peering TORs.
+2. Inter-subnet communication: Host routing enables the inter-subnet / routing capability of a local host with remote host which are in a different subnet. By advertising a more specific prefix than usual gateway address (SVI-IP), remote TORs know exactly where a host resides on the network. Optimal forwarding can always be achieved.
+FRR has the capability to synchronize MAC FDB and SVI-IP adjacency between peering PEs. ARP/ND adjacency prefixes are programmed in RIB/FIB on remote TOR.
+
+<a id=""></a>
+### 2.2.8 Mutual exclusion: MCLAG and EVPN MH
+The MCLAG configuration will not be allowed if ESI is configured on one or more interface(s).
+
+<a id=""></a>
+### 2.2.9 ARP and ND suppression
+ARP/ND suppression will be supported in EVPN Multihoming scenarios. The VTEP will respond to the ARP/ND requests received on local access ports only for the ARP/ND installed against remote VTEPs. ARP/ND response will not be generated by the VTEP if ARP/ND is synchronized and installed on the local Multihomed ES, even though the ARP/ND learning happened on the remote VTEP multihoming a given ES.
+
+<a id="3-Design"></a>
+# 3 Design
+
+<a id="31-Overview"></a>
+## 3.1 Overview
+The diagram below shows the flow of various EVPN MH specific events and messages between the components in SONiC subsystem.
+![image](https://github.com/hasan-brcm/SONiC/assets/56742004/3ae593a7-f3a9-4350-869b-dd7ad76b1ccb)
+
+
+- The below diagram summarizes the BUM traffic handling for a given ES:
+  
+![image](https://github.com/hasan-brcm/SONiC/assets/56742004/eb1eaff2-b991-42b1-b998-ff7aefc5604b)
+
+o EVPN_SPLIT_HORIZON_TABLE specifies the list of VTEPs for a given ES.
+
+o EVPN_DF_TABLE specifies the DF role for a given ES.
+<a id="32-DB-Changes"></a>
+## 3.2 DB Changes
+
+<a id="321-CONFIG-DB"></a>
+### 3.2.1 CONFIG DB
+
+**EVPN_ETHERNET_SEGMENT**
+
+```
+;New table
+;Specifies EVPN Ethernet Segment interface
+;
+; Status: stable
+key = EVPN_ETHERNET_SEGMENT|"ifname"if_id
```

Comment:
"ifname"if_id looks weird, can we just have an ifname as key and in the description, it's already mentioned about if_id.

Reply:
@venkatmahalingam , this is the convention that has been followed. "ifname"if_id means, "Ethernet"+0,1,2.. or "PortChannel"+1,2,3.. Same goes with Vlan.